src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -81,12 +81,6 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
       return;
     }
 
-    // "flutter test" prints this when using --start-paused.
-    // (We should probably change it to stop printing it in machine mode.)
-    if (trimmed.startsWith("You should first set appropriate breakpoints, then resume the test in the debugger.")) {
-      return;
-    }
-
```

Comment:
The test runs and output appears in the console, but it the Test UI claims that there was a tool error since it didn't see any progress events.

But I'm not sure if the `--machine` flag made it to the alpha branch either, in which case testing wouldn't work at all. Good point; we should make sure these two changes land in alpha before releasing the plugin.